src/start.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import datetime
+import logging
+import os
+import os.path
+import sys
+
+
+class Launcher:
+    def __init__(self):
+        self.logger = ...
+        self.file_handler = ...
+        self.init_logger()
+        self.logger.info("Starting server...")
+
+    def init_logger(self):
+        self.logger = logging.getLogger(__name__)
+        self.logger.setLevel(logging.DEBUG)
+
+        formatter = logging.Formatter("(%(asctime)s.%(msecs)d) [%(levelname)s]: \"%(name)s\": %(message)s",
+                                      "%d-%m-%Y %H:%M:%S")
+        console_handler = logging.StreamHandler()
+        console_handler.setFormatter(formatter)
+        console_handler.setLevel(logging.WARNING)
+        if not os.path.exists("log/"):
```

Comment:
Why do you create the logs directory dynamically, do you need a defensive code?

Reply:
The log/ folder doesn't exist on the repo (because I think it would just be clutter, plus it's ignored on the .gitignore file), so I need to create it, at least, on the first run on a new machine